yabause/src/sys/vdp1/src/vdp1.c

```diff
@@ -82,7 +82,7 @@ static void checkFBSync();
 
 int CONVERTCMD(s32 *A) {
   s32 toto = (*A);
-  if ((((*A)>>12)&0x1)^(((*A)>>11)&0x1) != 0) {
+  if ((*A>>12 & 0x1  ^  *A>>11 & 0x1) != 0) {
```

Comment:
gcc does not like this. I would ghave prefer to get a big parenthesis around. 
Something like:
(((((*A)>>12)&0x1)^(((*A)>>11)&0x1)) != 0)

Reply:
Which gcc version and which flag triggers it? I can't reproduce the complaint with either 13 or 14.